src/main/webview/event/eventManager.ts

```diff
@@ -5,16 +5,42 @@ import * as vscode from 'vscode';
 import { LogManager } from '../../log/logManager';
 import { LoginTask } from './tasks/login';
 import { JumpToCodeTask } from './tasks/jumpToCode';
+import { RunUtils } from '../../utils/runUtils';
 
 /**
  * Manages events and communication between the extension and the webview.
  */
 export class EventManager {
     protected send: EventSender;
+    private webviewLoaded: boolean = false;
+    private static TIMEOUT_THREE_MINUTES_IN_MS: number = 3 * 60 * 1000;
```

Comment:
There is no point in mentioning the timeout value in the key
```suggestion
    private static TIMEOUT_MILLISECS: number = 3 * 60 * 1000;
```